pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -69,29 +69,38 @@ private class UpdateConfigurationCmd extends CliCommand {
         @Parameter(names = {"-v", "--value"}, description = "service-configuration value", required = true)
         private String configValue;
 
+        @Parameter(names = {"-s", "--scope"}, description = "cluster or broker address, default is cluster",
```

Comment:
Shall we use an enum `Scope` instead of String here? Ditto other occurance.

Reply:
Scope is either cluster or broker address。 unable to enumerate